sql-cli/sql_cli/__main__.py

```diff
@@ -37,111 +41,105 @@ def about() -> None:
+        help="(Optional) Environment used to declare the connections to be validated",
     ),
-    connection: str = typer.Argument(
+    connection: str = typer.Option(
         default=None,
-        help="airflow connection id to validate",
+        help="(Optional) Identifier of the connection to be validated. By default checks all the env connections.",
     ),
 ) -> None:
-    validate_connections(environment=environment, connection_id=connection)
+    project_dir_absolute = project_dir.resolve() if project_dir else Path.cwd()
+    project = Project(project_dir_absolute)
+
+    validate_connections(project=project, environment=env, connection_id=connection)
 
 
 @app.command(
     help="""
     Run a workflow locally. This task assumes that there is a local airflow DB (can be a SQLite file), that has been
-    initialized with Airflow tables. Users can also add paths to a connections or variable yaml file which will override
-    existing connections for the test run.
-
-    \b
-    Example of a connections.yaml file:
-
-    \b
-    my_sqlite_conn:
-        conn_id: my_sqlite_conn
-        conn_type: sqlite
-        host: ...
-    my_postgres_conn:
-        conn_id: my_postgres_conn
-        conn_type: postgres
-        ...
+    initialized with Airflow tables.
     """
 )
 def run(
-    directory: Path = typer.Argument(
+    workflow_name: str = typer.Argument(
         default=...,
         show_default=False,
-        exists=True,
-        help="directory containing the sql files.",
+        help="name of the workflow directory within workflows directory.",
     ),
-    dags_directory: Path = typer.Argument(
-        default=...,
-        show_default=False,
-        exists=True,
-        help="directory for the generated DAGs.",
-    ),
-    connection_file: Path = typer.Option(
-        default=None,
-        exists=True,
-        help="path to connections yaml or json file",
+    env: str = typer.Option(
+        metavar="environment",
+        default="default",
+        help="environment to run in",
     ),
-    variable_file: Path = typer.Option(
-        default=None,
-        exists=True,
-        help="path to variables yaml or json file",
+    project_dir: Path = typer.Option(
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
     ),
+    verbose: bool = typer.Option(False, help="Whether to show airflow logs", show_default=True),
 ) -> None:
-    dag_file = generate_dag(directory, dags_directory=dags_directory)
-    dag = get_dag(dag_id=directory.name, subdir=dag_file.parent.as_posix())
-    run_dag(
-        dag=dag,
-        conn_file_path=connection_file.as_posix() if connection_file else None,
-        variable_file_path=variable_file.as_posix() if variable_file else None,
+    project_dir_absolute = project_dir.resolve() if project_dir else Path.cwd()
+    project = Project(project_dir_absolute)
+    project.update_config(environment=env)
```

Comment:
It feels a bit awkward that `run` is updating the project config - but we can address this in a separate PR.
If we remove this, the tests fail.